tmt/steps/execute/__init__.py

```diff
@@ -598,9 +598,12 @@ def summary(self) -> None:
 
         self.info('summary', ', '.join(message), 'green', shift=1)
 
-    def go(self) -> None:
+    def go(self, force: bool = False) -> None:
         """ Execute tests """
-        super().go()
+        super().go(force=force)
+
+        # Clean up possible old results
+        self._results.clear()
```

Comment:
We really need to refactor the `execute` step and plugins to **return** results rather than store them in a mutable list...

Reply:
Agreed, filed #2533 to track this.